terraform/deployments/cluster-services/logging.tf

```
@@ -34,6 +34,21 @@ resource "helm_release" "filebeat" {
           "loadbalance" : true
           "ssl.enabled" : true
         }
+        "logging.files" : {
+          "keepfiles" : 7
+        }
```

I think we want it not to log to the filesystem at all, no? Just stderr.
```suggestion
        "logging.to_files" : false
        "logging.to_stderr" true
```